### ISSUE TITLE
degree field and degree grantors field cleanup

### ISSUE THREAD

**Issue by revgum** [osulp/Scholars-Archive#929]
These two fields became "URI Enabled" recently, and have text values in Fedora and SOLR that aren't URIs. We need to find all of the works affected, lookup the appropriate URI for degree grantor and degree field, update Fedora, and reindex the work.

[WorksNeedingUpdates.zip](https://github.com/osulp/Scholars-Archive/files/1334171/WorksNeedingUpdates.zip)

**Comment by luisgreg99** [comment]
After cleaning up a few works so far in the production server, I found some `degree_grantors` values showing as `Oregon State College`. @vantuyls mentioned that it was an old name for the institution and that it should have its own URI. @wickr do we have a URI for this name in opaquenamespace?

**Comment by wickr** [comment]
@luisgreg99 You can use:
Oregon State College:
	http://id.loc.gov/authorities/names/n82022628
"Use this heading for items published by the college ca. 1937-3/6/1961"

**Comment by luisgreg99** [comment]
@wickr @vantuyls I found some degree_fields that didn't match exactly with the current collection of degree_fields http://opaquenamespace.org/ns/osuDegreeFields during the cleanup.

I made this spreadsheet to keep track of these fields: https://docs.google.com/a/oregonstate.edu/spreadsheets/d/1Wmm72rgnIT5tyrusxrWGjMTqcbhD4sXsWs27G9kJJ64/edit?usp=sharing

I believe some degree_field values are typos, so I attempted to find a close match (see close match1 and close match2 columns). Can I confirm that these matches are correct?

**Comment by wickr** [comment]
@luisgreg99 based on the values there the close match1s look good. For the ones with a close match2, I think the exact correctness will depend on if the date(s) of the items match the date ranges of the degrees.

**Comment by vantuyls** [comment]
finished work in the spreadsheet @luisgreg99. let me know if you have any questions.

**Comment by vantuyls** [comment]
@luisgreg99 are we good on this?> can we close the issue?

**Comment by luisgreg99** [comment]
@wickr @vantuyls yes, I just need to run my script to fix the uris provided for `degree_fields`. I will close it once I finish the task today. 

Also, I noticed that we are still missing the uris for these `degree_grantors`:
```
Oregon Agricultural College
Oregon State Agricultural College
Oregon State College in Chemistry
```
Can we create uris for these as well? I listed the works that use them in the tab `degree_grantors cleanup` at https://docs.google.com/spreadsheets/d/1Wmm72rgnIT5tyrusxrWGjMTqcbhD4sXsWs27G9kJJ64/edit#gid=383733877
